components/BlobMedia.tsx

```diff
@@ -8,7 +8,7 @@ import { encodeBase64 } from "../lib/storage/encoding";
 // @ts-ignore
 import { Image } from "native-base";
 
-export default function BlobMedia({ handle }: { handle: Handle }) {
+export function useMedia(handle: Handle): string | null {
```

Comment:
This hook should probably be pulled out into its own file, in `./hooks`, for consistency. It's not strictly part of the `BlobMedia` component here